mautic/mautic
# Focus/multipageform

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2992
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Multipage forms where shown all at once when inserted inside a focus item
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a multipage form inside Component/Form
2. Create a focus item which includes the multipage form created at step 1

#### Steps to test this PR:
1. Create a multipage form inside Component/Form
2. Create a focus item which includes the multipage form created at step 1. Now the form is displayed properly with the  pagination buttons

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
